### PR TITLE
Support AppInsights analytics & update to SignalR 2.2 nightly

### PR DESCRIPTION
**Pull request by DamianEdwards**
Add support for configuring an AppInsights ID for analytics
Updated to latest SignalR nightly build

### CODE REVIEW

**Comment by hnrkndrssn** [comment]
Looks good to me! :+1:

File: JabbR.Tests/app.config [hunk 1]
```diff
@@ -18,6 +18,14 @@
         <assemblyIdentity name="Iesi.Collections" publicKeyToken="aa95f207798dfdb4" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.Data.OData" publicKeyToken="31bf3856ad364e35" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-5.6.0.0" newVersion="5.6.0.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.AspNet.SignalR.Core" publicKeyToken="31bf3856ad364e35" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-2.1.0.0" newVersion="2.1.0.0" />
```
Comment (davidfowl) [inline JabbR.Tests/app.config:27]: This looks pretty wrong

File: JabbR/Web.config [hunk 2]
```diff
@@ -89,6 +89,10 @@
         <assemblyIdentity name="AjaxMin" publicKeyToken="21ef50ce11b5d80f" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-4.97.4951.28478" newVersion="4.97.4951.28478" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.AspNet.SignalR.Core" publicKeyToken="31bf3856ad364e35" culture="neutral" />
```
Comment (davidfowl) [inline JabbR/Web.config:93]: Why not update the whole stack?

File: JabbR.Tests/app.config (hunk 1 above)
Comment (DamianEdwards) [inline JabbR.Tests/app.config:27]: This of course was NuGet, not me. All I did was update SignalR.

File: JabbR/Web.config (hunk 2 above)
Comment (DamianEdwards) [inline JabbR/Web.config:93]: I don't follow. I updated Microsoft.AspNet.SignalR. What do you mean by whole stack?

Comment (davidfowl) [inline JabbR/Web.config:93]: All of the signalr packages so that we don't end up with binding redirects for signalr

Comment (DamianEdwards) [inline JabbR/Web.config:93]: Ah you mean I forgot the scale-out packages :)

**Pull request merged**